permissions/plugin-katalon.yml

```
@@ -6,4 +6,4 @@ issues:
 paths:
 - "org/jenkins-ci/plugins/katalon"
 developers:
-- "devalex88"
+- "hainguyenkatalon,duyluonganh,HieuBui419"
```

this is a yaml list, so should be
```suggestion
- "hainguyenkatalon"
- "duyluonganh"
- "HieuBui419"
```